admin/resources/school/SchoolCreate.tsx

```diff
@@ -103,28 +167,32 @@ const SchoolCreate = () => {
     }).sort((a: any, b: any) => { return a.name < b.name ? -1 : 1 });;
   }, [selectedBlock, districtData]);
 
+  console.log(data)
+
 
   return (
-    <Create mutationOptions={{ onSuccess }}>
+    <Create mutationOptions={{ onSuccess, onError }}>
       <SimpleForm>
-        <TextInput source="name" validate={inputConstraints.fullName} />
-        <NumberInput source="udise" validate={required("Please enter a valid UDISE")} />
-        {/* <SelectInput label="District" source="location.data.district" onChange={(e: any) => {
+        <TextInput source="name" validate={inputConstraints.fullName} onChange={e => setData((prevData: any) => { return { ...prevData, name: e.target.value } })} />
+        <NumberInput source="udise" validate={required("Please enter a valid UDISE")} onChange={e => setData((prevData: any) => { return { ...prevData, udise: e.target.value } })} />
```

Comment:
it's correct but can we written in more clear way like 
`onChange={e => setData((prevData: ProperType) => ({ ...prevData, name: e.target.value })} `

Reply:
okay